frontend/public/style/_vars.scss

```diff
@@ -19,26 +19,13 @@ $fa-font-path: $consoleWebDistPath + '/';
 // Use PatternFly 4 font family in PatternFly 3.
 $font-family-base: var(--pf-global--FontFamily--sans-serif);
 $font-size-base: 14px;
-// Use PatternFly 4 heading font sizes in Patternfly 3.
-$font-size-h1: var(--pf-global--FontSize--2xl);
-$font-size-h2: 20px; // must be literal value because of calculation in patternfly/dist/sass/patternfly/_list-view.scss
-$font-size-h3: var(--pf-global--FontSize--lg);
-$font-size-h4: var(--pf-global--FontSize--md);
-$font-size-h5: var(--pf-global--FontSize--md);
-$font-size-h6: var(--pf-global--FontSize--md);
-// Patternfly 3 defaults to 40px, which leaves a lot more whitespace than we're used to.
 $grid-gutter-width: 30px;
-// Use PatternFly 4 headings font styles in PatternFly 3.
-$headings-font-family: var(--pf-global--FontFamily--heading--sans-serif);
-$headings-font-weight: var(--pf-global--FontWeight--normal);
-$kbd-bg: transparent;
-$kbd-color: $color-pf-black-600;
 // Match Patternfly 4 link colors
 $link-color: var(--pf-global--link--Color);
 $link-hover-color: var(--pf-global--link--Color--hover);
 // Disable PatternFly 4 reset.
-$pf-global--enable-reset: false;
-$pf-global--enable-shield: true;
+$pf-global--enable-reset: true;
+$pf-global--enable-shield: false;
```

Comment:
This is the default value, so you can remove the var: https://github.com/patternfly/patternfly/blob/7842e225de1884a41160551a890eb6cb3f4541b8/src/patternfly/sass-utilities/scss-variables.scss#L15

Reply:
removed